packages/contentful/migrations/gp2/outputs/20230801150001-add-keywords.js

```diff
@@ -0,0 +1,30 @@
+module.exports.description = 'Add tags to outputs content model';
+
+module.exports.up = (migration) => {
+  const outputs = migration.editContentType('outputs');
+
+  outputs
+    .createField('tags')
```

Comment:
The name tags is not consistent with the field name in Events.

Should there be a min/max number added?

Reply:
Yes, because in Projects, WG, outputs and users the existing field is keywords, and in events is tags. So, to complete the migration, we needed to create another field with a different name since we need to keep them both until it's done. We can align on one single name to keep consistency for the last PR.
Regarding validations, we copied the validations that were in the CMS previously